### ISSUE TITLE
Problem with http-interop/http-middleware dependencie

### ISSUE THREAD

**Issue by up2dev** [webimpress/http-middleware-compatibility#5]
It seems that this dependencie is deprecated.
the owner recommend to require "psr/http-server-middleware"

**Comment by michalbundyra** [comment]
@up2dev Yes, this repository is deprecated now. PSR-15 landed last week and I would suggest to use it. I'll make further changes in the repository later on. Thanks for reporting.
